### PR TITLE
Add status bot

### PR DESCRIPTION
**Pull request by penalosa**
## What this PR solves / how to test

Migrates the ANT status bot to GitHub

## Author has addressed the following

- Tests
  - [ ] TODO (before merge)
  - [ ] Included
  - [x] Not necessary because: this code is not currently tested
- E2E Tests CI Job required? (Use "e2e" label or ask maintainer to run separately)
  - [ ] I don't know
  - [ ] Required / Maybe required
  - [x] Not required because: 
- Changeset ([Changeset guidelines](https://github.com/cloudflare/workers-sdk/blob/main/CONTRIBUTING.md#changesets))
  - [ ] TODO (before merge)
  - [x] Included
  - [ ] Not necessary because:
- Public documentation
  - [ ] TODO (before merge)
  - [ ] Cloudflare docs PR(s): <https://github.com/cloudflare/cloudflare-docs/pull/>...
  - [x] Not necessary because: internal only

### CODE REVIEW

File: packages/devprod-status-bot/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,571 @@
...
+		"https://api.github.com/repos/cloudflare/workers-sdk/pulls?state=open&per_page=100",
+		{
+			headers: {
+				"User-Agent": "Cloudflare ANT Status bot",
+				Authorization: `Bearer ${pat}`,
+			},
+		}
+	).then((r) => r.json<PRList>());
+	const miniflare = await fetch(
+		"https://api.github.com/repos/cloudflare/miniflare/pulls?state=open&per_page=100",
```
Comment (Skye-31) [inline packages/devprod-status-bot/src/index.ts:40]: Probably not relevant anymore

File: packages/devprod-status-bot/package.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+	"name": "devprod-status-bot",
+	"version": "0.0.0",
+	"private": true,
+	"scripts": {
+		"deploy": "wrangler deploy"
```
Comment (petebacondarwin) [inline packages/devprod-status-bot/package.json:6]: Should we also do eslint checking here?

```
"check:lint": "eslint .",
```

**Review by petebacondarwin — APPROVED** [review]
I assume this has been prettify formatted?

File: packages/devprod-status-bot/package.json (hunk 2 above)
Comment (petebacondarwin) [reply packages/devprod-status-bot/package.json:6]: And maybe:

```
"check:type": "tsc"
```

**Pull request merged**